doc/release/new-policy.rst

```diff
@@ -0,0 +1,341 @@
+        and collect feedback before an upcoming release.
+        Release candidate has the same feature set as the preceding beta version
+        and doesn't have known bugs in typical usage scenarios
+        or degradations from the previous release series.
+
+        Release candidate is a good choice to set up a staging server.
+
+        ..  code-block:: text
+
+            MAJOR.MINOR.PATCH-rcN
+
+            3.0.0-rc1
+            3.0.0-rc2
+            3.0.1-rc1
+
+Release series lifecycle
+--------------------------
+
+A release series goes through the following stages:
+
+..  contents::
+    :local:
+
+Early development
+~~~~~~~~~~~~~~~~~
+
+The early development stage goes on until the first :term:`major release <major release>`.
+Alpha, beta, and release candidate versions are published within this stage.
+
+The stage splits into two phases:
+
+1.  Development of a new functionality through alpha and beta versions.
+    Features can be added and, sometimes, removed in this phase.
+
+2.  Stabilization starts with the first release candidate version.
+    Feature set doesn't change in this phase.
+
+Support
+~~~~~~~
+
+The stage starts when a first release is published.
+The release series now is an object of only backward compatible changes.
+
+At this stage, all known security problems and all found
+degradations since the previous series are fixed.
+
+A series receives degradation fixes and other bugfixes during the support stage
+and until the series transitions into the end of life (EOL) stage.
+
+The decision of whether to fix a particular problem in a particular release series
+depends on the impact of the problem, risks around backward compatibility and the
+complexity of backporting a fix.
+
+A release series might receive new features at this stage,
+but only in a backward compatible manner.
+A release candidate might be published for a new functionality before a release.
+
+During the support period a release series receives new versions of supported Linux
+distributives to build infrastructure.
+
+The intended duration of the support period for each series is at least two years.
+
+End of life
+~~~~~~~~~~~
+
+A series reaches the end of life (EOL) when the last release in the series is
+published. The series will not receive updates anymore.
+
+In modules, connectors and tools, we don't guarantee support of a release series
+that reaches EOL.
+
+A release series cannot reach EOL until the vast majority of production environments,
+for which we have commitments and SLAs, will be updated to a newer series.
+
+
+Versions per lifecycle stage
+----------------------------
+
+..  container:: table
+
+    ..  rst-class:: left-align-column-1
+    ..  rst-class:: left-align-column-2
+
+    ..  list-table::
+        :header-rows: 1
+
+        *   -   Stage
+            -   Version types
+            -   Examples
+
+        *   -   Early development
+            -   Alpha, beta, release candidate, nightly builds (not published)
```

Comment:
I doubt we should menting development builds in this table. It is not something published for a user. Similarly, we don't mention a custon builds from developer branches here.